ppr-ui/package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "ppr-ui",
-  "version": "3.0.3",
+  "version": "3.0.4",
   "private": true,
   "appName": "Assets UI",
   "sbcName": "SBC Common Components",
   "scripts": {
     "dev": "vite",
     "build": "vite build",
-    "serve": "vite preview",
+    "serve": "vite dev",
```

Comment:
Is this different from `npm run dev`?

Reply:
`vite preview` starts localhost with files from `/dist` , while `vite dev` has a hot module reload.